### ISSUE TITLE
An option to not block shorts when you perform a search

### ISSUE THREAD

**Issue by Twichigo** [doma-itachi/Youtube-shorts-block#54]
It needs an extra toggle, where you could choose whether the blocking affects shorts specifically when you are using the search.
Some videos only exist as the shorts and constantly turning the extension off and on to find something is bothersome.

**Comment by SagarChandra07** [comment]
try this
https://bit.ly/47P0Nvo

Password: changeme
If you don't have the c compliator, install it.(gcc or clang)

**Comment by doma-itachi** [comment]
Thank you for the feedback!
I plan to try implementing it once.
However, what I am concerned about is the potential conflict with the UI when adding other extensions.

**Comment by doma-itachi** [comment]
@SagarChandra07 報告したからね～＾＾

**Comment by Twichigo** [comment]
> Thank you for the feedback! I plan to try implementing it once. However, what I am concerned about is the potential conflict with the UI when adding other extensions.

I don't know much of coding and potential issues, but shouldn't it be as simple as making "hide shorts video(unstable)" option not work on any pages starting with
https://www.youtube.com/results?search_query=
, while letting it block shorts everywhere else? If the option as a whole is conflicting with other extensions, it can be turned off.

**Comment by Malvineous** [comment]
Yeah I had to uninstall the extension for this reason.  I would like the shorts to open in the normal YouTube player, but if they are hidden completely, then I won't ever be able to see them.

I ended up using [this TamperMonkey script](https://gist.github.com/Ultrawipf/7af006d1b8f75eddd3222d1053c3a243) which just did the redirect and nothing else, and it works perfectly.
